ramls/transaction-summary.raml

```diff
@@ -1,7 +1,7 @@
 #%RAML 1.0
 title: "mod-finance-storage"
 baseUri: https://github.com/folio-org/mod-finance-storage
-version: v1
+version: v2
```

Comment:
There was no major changes in transaction summary API. So I think version is supposed to be v1.1

Reply:
fixed